xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -350,20 +458,80 @@ viewers.Comments = Class.create({
       this.initialLocation.insert({after: this.form});
     }
     this.form["XWiki.XWikiComments_replyto"].value = "";
-    this.form["XWiki.XWikiComments_comment"].value = "";
+    this.reloadEditor();
     this.cancelPreview(this.form);
   },
   /**
    * Registers a listener that watches for the insertion of the Comments tab and triggers the enhancements.
    * After that, the listener removes itself, since it is no longer needed.
    */
-  addTabLoadListener : function(event) {
-    var listener = function(event) {
+  addTabLoadListener: function (event) {
+    var listener = function (event) {
       if (event.memo.id == 'Comments') {
         this.startup();
       }
     }.bindAsEventListener(this);
     document.observe("xwiki:docextra:loaded", listener);
+  },
+  destroyEditor: function (selector, name) {
+    require(['jquery', 'xwiki-events-bridge'], function ($) {
+      $(document).trigger('xwiki:actions:cancel');
+      if ($(selector).size() > 0) {
+        const parent = $(selector).parent(".wysiwyg-field");
+        parent.empty();
+        $(document).trigger('xwiki:dom:updated', {'elements': parent.toArray()});
+      }
+    });
+  },
+  reloadEditor: function (options) {
```

Comment:
Not sure about the name: it's actually a `loadEditor` more than a "re" load for me...